fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTray.kt

```diff
@@ -71,6 +75,7 @@ fun TabsTray(
     browserStore: BrowserStore,
     tabsTrayStore: TabsTrayStore,
     displayTabsInGrid: Boolean,
+    taskContinuityEnabled: Boolean,
```

Comment:
Tangential, but do we know the status of this flag? I would maybe assume it's safe to remove by now. Is there a ticket tracking it or anything?

Reply:
As far as I know, this is a zombie secret setting. It was enabled by default in last September, but we don't have a ticket to clean it up.

Reply:
Let's not make more work here by passing around this flag. You could just hardcode true in the `SyncedTabsList` and file a cleanup issue for the backlog.

Is there a reason to propagate this all the way up to the `TabsTray` level?

Reply:
Passing in true down to `SyncedTabsList` could make sense.

If being derived from Settings, the flags have to propagate from the fragment + `TabsTray` in order to not break the previews. (calls to `Components` break the preview compilation)

Reply:
I'm not sure I follow, sorry. I was suggesting: removing this flag from this line and making it `true` on line 211 below.

Does this have an effect on previews?

Reply:
that made sense and I'm pushing that now - and no that doesn't break previews.

Sorry for the confusion. I was more trying to explain that having this flag as a parameter in `TabsTray` was just to abstract `Settings` (and therefore `Components`) away from the Compose code.

Reply:
Sounds good! 

> we don't have a ticket to clean it up.

Did you already file a bug for this? It would be a good-first-bug for an external contributor but also something we can pick up during some downtime.

Reply:
File a bug and add the `good-first-bug` keyword.

Reply:
Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1829018